sam/sim/src/rd_scanner.py

```diff
@@ -169,23 +166,24 @@ def __init__(self, crd_arr=[], seg_arr=[], skip=True, **kwargs):
         self.curr_ref = ''
         self.curr_crd = ''
         self.emit_fiber_stkn = False
-
         self.meta_clen = len(crd_arr)
         self.meta_slen = len(seg_arr)
 
         # Statistics
-        self.unique_refs = []
-        self.unique_crds = []
-        self.total_outputs = 0
-        self.elements_skipped = 0
-        self.skip_cnt = 0
-        self.intersection_behind_cnt = 0
-        self.fiber_behind_cnt = 0
+        if self.get_stats:
+            self.unique_refs = []
+            self.unique_crds = []
+            self.total_outputs = 0
+            self.elements_skipped = 0
+            self.skip_cnt = 0
+            self.intersection_behind_cnt = 0
+            self.fiber_behind_cnt = 0
+        self.stop_count = 0
```

Comment:
I think this should be under self.get_stats?

Reply:
You mean stop_count = 0 line? I'll correct that. 
Rest of it is gaurded I hope./

Reply:
@Ritvik1sharma the stop_count from what i've seen is only used in the dictionary

Reply:
Yes I got that. I had corrected this in the pull request.